Misc/NEWS.d/next/Library/2020-09-10-07-23-24.bpo-41730.DyKFi9.rst

```diff
@@ -0,0 +1 @@
+The ``tkinter.tix`` module has been deprecated since Python 3.6. Warn a ``DeprecationWarning`` when importing it.
```

Comment:
```suggestion
``DeprecationWarning`` is now raised when importing :mod:`tkinter.tix`, which has been deprecated in documentation since Python 3.6.
```